app/src/deadlink.php

```diff
@@ -14,9 +14,15 @@
 	You should have received a copy of the GNU Affero General Public License
 	along with this program.  If not, see <https://www.gnu.org/licenses/agpl-3.0.html>.
 */
+$namespace = 0;
 if( !empty( $argv[1] ) ) {
-	echo "Set to run on {$argv[1]}\n";
-	define( 'WIKIPEDIA', $argv[1] );
+	$parts = explode(':', $argv[1]);
+	echo "Set to run on {$parts[0]}\n";
+	define( 'WIKIPEDIA', $parts[0] );
+	if ( $parts[1] ) {
```

Comment:
This will throw a warning if undefined.  I think you want if( !empty( $parts[1] ) ).